src/Commands/SixMans.py

```diff
@@ -286,134 +350,204 @@ def brokenQueue(player: Member) -> Embed:
+            embed = InfoEmbed(
+                title="Leaderboard Placement for {0}".format(member["Name"].split("#")[0]),
                 desc=players_rank
             )
+            edited_embed = CaptainsRandomHelpEmbed(embed, blueTeam, orangeTeam, blueCap, orangeCap)
+            return edited_embed
 
-        return ErrorEmbed(
+        embed = ErrorEmbed(
             title="No Matches Played",
-            desc="{0} hasn't played any matches and won't show up on the leaderboard.".format(players_rank.mention)
+            desc="{0} hasn't played any matches and won't show up on the leaderboard.".format(arg[0])
         )
+        edited_embed = CaptainsRandomHelpEmbed(embed, blueTeam, orangeTeam, blueCap, orangeCap)
+        return edited_embed
 
-    if (len(arg) == 0 and len(mentions) == 0):
+    elif (len(arg) == 0 and playerMentioned == False):
         viewFullLb = "\nTo see the full leaderboard, visit <#{0}>.".format(lbChannelId) if (lbChannelId != -1) else ""
-        return InfoEmbed(
+        embed = InfoEmbed(
             title="UNCC 6 Mans | Top 5",
             desc=Leaderboard.showLeaderboard(limit=5) + viewFullLb
         )
-
-    return ErrorEmbed(
-        title="Leaderboard Command Help",
-        desc="Mention someone to see their rank, use 'me' to see your rank,"
-        " include nothing to see the top 5 on the leaderboard."
-    )
+        edited_embed = CaptainsRandomHelpEmbed(embed, blueTeam, orangeTeam, blueCap, orangeCap)
+        return edited_embed
+
+    else:
+        embed = ErrorEmbed(
+            title="Leaderboard Command Help",
+            desc="Mention someone to see their rank, use 'me' to see your rank,"
+            " include nothing to see the top 5 on the leaderboard."
+        )
+        edited_embed = CaptainsRandomHelpEmbed(embed, blueTeam, orangeTeam, blueCap, orangeCap)
+        return edited_embed
 
 
-async def report(player: Member, lbChannel: Channel, *arg) -> Embed:
+async def report(player: Member, lbChannel: Channel, winningTeam: Literal["blue", "orange"]) -> bool or None:
     """
         Used to report the winning team of the series.
 
         Parameters:
             player: discord.Member - The author of the message.
             lbChannel: discord.Channel - The specified leaderboard channel object
+            winningTeam: str - The reported winning team
 
         Returns:
-            dicord.Embed - The embedded message to respond with.
+            dicord.Embed - The embedded message to respond with or sends nothing.
     """
-    if (len(arg) == 1 and (str(arg[0]).lower() == Team.BLUE or str(arg[0]).lower() == Team.ORANGE)):
-        msg = Leaderboard.reportMatch(player, arg[0])
-
-        if (":x:" in msg):
-            return ErrorEmbed(
-                title="Match Not Found",
-                desc=msg[4:]
-            )
-        if (":white_check_mark:" in msg):
+    msg = Leaderboard.reportMatch(player, winningTeam)
```

Comment:
Isn't `msg` here a bool now? If so, I would rename it to something like `match_reported` or something to better indicate that it's a boolean.

Reply:
Resolved.